src/systems/system.C

```diff
@@ -1462,6 +1464,15 @@ Real System::calculate_norm(const NumericVector<Number> & v,
       return v_norm;
     }
 
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+
+  // One way for implementing this would be to exchange the fe with the FEInterface- class.
+  // However, it needs to be discussed whether integral-norms make sense for infinite elements.
+  // or in which sense they could make sense.
+  libmesh_not_implemented();
```

Comment:
Oh, absolutely *not*.  IIRC I always enable infinite elements at configure time indirectly via --enable-everything, and that doesn't mean I want to break any code that tries to calculate a norm on a mesh which doesn't include them.

L-infinity norm at least ought to make sense with infinite elements in the mesh, right?  But even if you don't want to implement that, at least move the ```libmesh_not_implemented()``` into the element loop where you can check ```elem->infinite()``` first.

Reply:
yes, you are absolutely right.